store/extensions/G-WallMover/extension.json

```
@@ -32,7 +32,7 @@
         "clients": ["Origins"]
     },
   
-    "submissionDate": "06-07-2024 23:45:00",
-    "updateDate": "06-07-2024 23:45:00",
+    "submissionDate": "09-07-2024 20:417:00",
+    "updateDate": "09-07-2024 20:417:00",

```

```suggestion
    "submissionDate": "06-07-2024 23:45:00",
    "updateDate": "09-07-2024 20:417:00",
```
Don't change the submissionDate only the updateDate, also 417?